src/Common/src/Interop/Windows/sspicli/SecuritySafeHandles.cs

```diff
@@ -872,7 +872,7 @@ public new IntPtr DangerousGetHandle()
                     GlobalLog.Assert("SafeDeleteContext::AcceptSecurityContext()|inSecBuffer == null || inSecBuffers == null");
                 }
 
-                Debug.Fail("SafeDeleteContext::AcceptSecurityContext()|outSecBuffer != null");
+                Debug.Fail("SafeDeleteContext::AcceptSecurityContext()|inSecBuffer == null || inSecBuffers == null");
```

Comment:
Ah, yes.  That's true.  I see that line 573 has this same message/if.  But 1151 prints the assert, instead of the negative-assert.

Given that this fix makes it 4 vs 1; I'll say LGTM.  If you want to wait for Cristian to chime in I won't fault you for it :)